src/plugins/intel_gpu/src/graph/impls/onednn/reorder_onednn.cpp

```diff
@@ -87,8 +87,8 @@ struct reorder_onednn : typed_primitive_onednn_impl<reorder, dnnl::reorder::prim
 
         const kernel_impl_params* impl_params = reinterpret_cast<kernel_impl_params*>(ib.getKernelImplParams());
 
-        auto input_md = onednn::layout_to_memory_desc(impl_params->get_input_layout(0));
-        auto output_md = onednn::layout_to_memory_desc(impl_params->get_output_layout());
+        auto input_md = onednn::layout_to_memory_desc(impl_params->get_input_layout(0), dnnl::memory::format_tag::undef, false, false, true);
+        auto output_md = onednn::layout_to_memory_desc(impl_params->get_output_layout(), dnnl::memory::format_tag::undef, false, false, true);
```

Comment:
Frankly, in Reorder perspective, too complicated parameters, even though no changes.
How about change parameter name from _need_blocked_ to _planar_format_ or _no_blocked_ or something similar thing in _layout_to_memory_desc_?